tests/unit/test_configuration_parser.py

```diff
@@ -124,7 +130,44 @@ def test_schema_verification(tmpdir, stub_creds, stub_configs):
     cred_file.write_text(json.dumps(stub_creds))
     replica_file.write_text(json.dumps(stub_configs))
 
-    replica_config = ConfigurationParser()._get_dict_from_anything(replica_file, REPLICA_JSON_SCHEMA)
-    cred_config = ConfigurationParser()._get_dict_from_anything(cred_file, CREDENTIALS_JSON_SCHEMA)
+    replica_config = ConfigurationParser()._get_dict_from_anything(
+        replica_file, REPLICA_JSON_SCHEMA)
+    cred_config = ConfigurationParser()._get_dict_from_anything(
+        cred_file, CREDENTIALS_JSON_SCHEMA)
     assert isinstance(replica_config, dict)
     assert isinstance(cred_config, dict)
+
+
+def test_materialization_mappings(stub_configs):
+    cases = {
+        'true': {
+            'value': True,
+            'result': {
+                "BASE TABLE": materializations.TABLE,
+                "VIEW": materializations.TABLE
+            }
+        },
+        'false': {
+            'value': False,
+            'result': {
+                "BASE TABLE": materializations.TABLE,
+                "VIEW": materializations.VIEW
+            }
+        },
+        'undefined': {
+            'value': None,
+            'result': {
+                "BASE TABLE": materializations.TABLE,
+                "VIEW": materializations.TABLE
+            }
+        }
+    }
+
+    for case in cases.values():
```

Comment:
the https://docs.pytest.org/en/6.2.x/parametrize.html is more flexible&convenient approach to use 
every single iteration is treated a sep test case